pkg/trace/api/info.go

```diff
@@ -59,12 +60,12 @@ func (r *HTTPReceiver) makeInfoHandler() http.HandlerFunc {
 		oconf.Memcached = o.Memcached.Enabled
 	}
 	txt, err := json.MarshalIndent(struct {
-		Version      string        `json:"version"`
-		GitCommit    string        `json:"git_commit"`
-		BuildDate    string        `json:"build_date"`
-		Endpoints    []string      `json:"endpoints"`
-		FeatureFlags []string      `json:"feature_flags,omitempty"`
-		Config       reducedConfig `json:"config"`
+		Version      string          `json:"version"`
+		GitCommit    string          `json:"git_commit"`
+		BuildDate    string          `json:"build_date"`
+		Endpoints    []string        `json:"endpoints"`
+		FeatureFlags map[string]bool `json:"feature_flags,omitempty"`
```

Comment:
Agent internal feature flags, configs can actually be removed. The "feaures" required for agent -> tracers are what could be used by the tracer:
- endpoints, agent version (commit/version), and [clientDropP0 property](https://github.com/DataDog/datadog-agent/blob/0eeb74ec5d67fd7e2c547a4557b14ce580662cb7/pkg/trace/api/info.go#L75) I'll add in another PR

Reply:
Are you sure? That would be better actually, to drop these out completely...

Reply:
Reverted this change and cleaned up a bit, removing more config...